changelog/updates/2022-01-09-weekly-update.md

```diff
@@ -0,0 +1,11 @@
+- XZ utils ([5.2.10](https://git.tukaani.org/?p=xz.git;a=blob;f=NEWS;h=d92fa88a835180af5d6ff22ad0e240d6468f81af;hb=f7c2cc55618b9af3318f0c908cf8db0df1e28e7c))
```

Comment:
Can you please rename `changelog/updates/2022-01-09-weekly-update.md` to `changelog/updates/2023-01-09-weekly-update.md`. (without s?)

Reply:
Done.

Reply:
Now with an s for consistency.